server/controllers/article.js

```diff
@@ -80,10 +82,44 @@ class Article {
       );
     }
 
+    let bookmarks;
     if (userId && userId !== article.userid) {
       await articleRepo.addReadingStat(userId, article.id);
+      const usersBookmarks = await bookmarkRepo.getBookmarkedArticles(userId);
+      bookmarks = usersBookmarks.bookmarks.map(element => element.dataValues);
     }
 
+    const love = article.Reactions.filter(reaction => reaction.reactionType === 'Love');
```

Comment:
```suggestion
    const REACTIONS = {
      LOVE: 'Love',
      . . .
    };
    . . .
    const love = article.Reactions.filter(reaction => reaction.reactionType === REACTIONS.LOVE);
```

You want to whine us @TheoOkafor
CC: @wiztemple @PriscillaSam @ucheg6